components/StyledHr.js

```diff
@@ -12,6 +12,12 @@ const StyledHr = styled.hr`
   margin: 0;
   height: 1px;
 
+  ${props =>
+    props.color &&
+    css({
+      'border-color': props.color.includes('#') ? `${props.color}` : themeGet(`colors.${props.color}`),
+    })}
+
```

Comment:
`StyledHr` already supports the `borderColor` prop from styled-system (so it supports theme values).

This check is also wrong, it will fail if you provide `color="rgb(10,10,10)"`